pandas/core/indexes/multi.py

```diff
@@ -2157,9 +2157,10 @@ def _drop_from_level(self, codes, level, errors="raise"):
         index = self.levels[i]
         values = index.get_indexer(codes)
 
+        not_found = np.array(codes)[np.array(values) == -1].tolist()
```

Comment:
That won't work after #37794 is merged. Could you rebase afterwards? Please check for len(not_found)

Reply:
ok

Reply:
is the to_list necessary?

Reply:
not necessary, I will alter this after phofl's PR merged.

Reply:
You could rebase now